indico/modules/events/editing/client/js/components/EditingView/EditingFooter.jsx

```diff
@@ -0,0 +1,46 @@
+// This file is part of Indico.
+// Copyright (C) 2002 - 2020 CERN
+//
+// Indico is free software; you can redistribute it and/or
+// modify it under the terms of the MIT License; see the
+// LICENSE file for more details.
+import logoURL from 'indico-url:assets.image';
+import contactURL from 'indico-url:core.contact';
+import tosURL from 'indico-url:legal.display_tos';
+
+import React from 'react';
+import {Image} from 'semantic-ui-react';
+import {Translate} from 'indico/react/i18n';
+
+import './EditingFooter.module.scss';
+
+export default function Editingfooter() {
```

Comment:
should be `EditingFooter` but do we really want to put the "base" part of this in the editing module?